adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/PlatformParameters.cs

```diff
@@ -98,6 +98,10 @@ public PlatformParameters(Activity callerActivity, bool useBroker, PromptBehavio
         /// </summary>
         public PromptBehavior PromptBehavior { get; set; }
 
+        /// <summary>
+        /// Used to set embedded 
+        /// ADAL only supports embedded webview.
```

Comment:
>ADAL only supports embedded webview. [](start = 12, length = 36)

Should perhaps be in the <remarks>section of the comments instead?

Reply:
done.

---
In reply to: [190771904](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1069#discussion_r190771904) [](ancestors = 190771904)